examples/plugins/ebpf/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM ubuntu:wily
```

Comment:
Might want to consider using [tinycore python](https://github.com/weaveworks/scope/blob/master/experimental/example/app/Dockerfile#L1) if possible, ubuntu is a rather large image.